README.md

```diff
@@ -20,6 +20,9 @@ The LogDNA AWS CloudWatch integration relies on [AWS Lambda](https://aws.amazon.
     * `LOGDNA_HOSTNAME`: Alternative Host Name *(Optional)*  
     * `LOGDNA_TAGS`: Comma-separated Tags *(Optional)*  
     * `LOGDNA_URL`: Custom Ingestion URL *(Optional)*
+    * `LOG_RAW_EVENT`: Setting `line` to Raw `event.message` *(Optional, Default: false)*:
+        * It can be enabled by setting `LOG_RAW_EVENT` to `YES` or `TRUE`
+        * Enabling it moves the following `event`-related `meta` data from the `line` field to the `meta` field
```

Comment:
@smusali "moves the following" is confusing here since you don't list any fields that get moved. Might be more accurate to say all metadata fields.

Reply:
@matt-march, done! Added!